airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/csv/StagingDatabaseCsvSheetGenerator.java

```diff
@@ -30,12 +29,16 @@
  */
 public class StagingDatabaseCsvSheetGenerator implements CsvSheetGenerator {
 
-  private final boolean use1s1t;
+  private final boolean useDestinationsV2Columns;
   private final List<String> header;
 
   public StagingDatabaseCsvSheetGenerator() {
-    use1s1t = TypingAndDedupingFlag.isDestinationV2();
-    this.header = use1s1t ? JavaBaseConstants.V2_RAW_TABLE_COLUMN_NAMES : JavaBaseConstants.LEGACY_RAW_TABLE_COLUMNS;
+    this(false);
```

Comment:
true?

Reply:
this is to maintain v1 behavior in old destinations (i.e. redshift I think?). V2 destinations are all using the new constructor and explicitly passing true.